model_fitting/create_heatmap.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+import seaborn as sb
+import matplotlib.pyplot as plt
+import pandas as pd
+from sklearn import preprocessing
+import sys
+import os
+
+
+
+name_test=['COBRA','naive','sichuan','swiss','texas']
+dic={'COBRA_col':['sample_name','CGPHSSLFSC', 'CGHSSLFSCLAC','CLHSSLFSSPCP','CLSYSSLFSC','CPHTSTIFSGPC','CPHHSSLFSC'],
+'naive_col':['sample_name','CLPQLPALSQCC', 'CGGPEWREKC','CGSNESREKC','CETDVTGLLINC'],
+'sichuan_col':['sample_name','SLLRLEPSS','CQIRAKDQGLVC','AASMIDKPT','LLLSLSAPCC','PYHMLHIWGVEP','SWYVETPIAK'],
+'swiss_col':['sample_name','CVTAVDPIHLTLMC','PPSLYARFD','TTPLGLFERF','PPSLYDRFSP'],
+'texas_col':['sample_name','CKLSSLFSSC','CKLSSLFSRC','CRDVSLFSSC','RKESSLFSVC','CKTINLFSRC']}
```

Comment:
In order for this code to be generic this should be an input.
Think about a simple format, maybe csv, where we can express the data we want to plot